src/Raven.Server/Documents/PeriodicBackup/BackupTask.cs

```diff
@@ -810,6 +912,12 @@ public Stream GetOutputStream(Stream fileStream)
 
         private void UploadToServer(string backupPath, string folderName, string fileName)
         {
+            if (_directUploadDestination != DirectUploadDestination.Disabled)
```

Comment:
Shouldn't it be checked by the caller of this method?

Reply:
changed